esp/templates/program/modules/surveymanagement/import.html

```diff
@@ -62,7 +62,7 @@ <h2>Confirm Survey Import</h2>
     {% for question in questions %}
         <tr>
             <td class="underline">{{ question.seq }}</td>
-            <td class="underline" colspan="4">{{ question.render }}</td>
+            <td class="underline" colspan="4">{% if question.per_class %}{{ question.render|substitute:section }}{% else %}{{ question.render|uselist:classes }}{% endif %}</td>
```

Comment:
I get this error when trying to import a survey:

![image](https://user-images.githubusercontent.com/25392075/89846400-22be0800-db4f-11ea-95c0-8c9bf355a2f8.png)